tests/tests.rs

```diff
@@ -1911,3 +1911,55 @@ fn serde() {
         "invalid value: integer `-1`, expected u128",
     );
 }
+
+#[cfg(all(feature = "borsh", feature = "std"))]
+#[test]
+fn borsh() {
+    use borsh::schema::BorshSchemaContainer;
+    use borsh::{BorshDeserialize, BorshSerialize};
+    let mut buf = Vec::new();
+    let input = u9::new(42);
+    input.serialize(&mut buf).unwrap();
+    let output = u9::deserialize(&mut buf.as_ref()).unwrap();
+    assert_eq!(buf.len(), (u9::BITS + 7) / 8);
+    assert_eq!(input, output);
+
+    let input = u63::MAX;
```

Comment:
that's not a very good test case I think. Maybe pick a number so that you can assert the buffer actually contains what we expect?